tests/issue0303.d

```diff
@@ -0,0 +1,4 @@
+static foreach (thing; things){doStuff();}
+static foreach_reverse (thing; things){doStuff();}
+static foreach (thing; things) doStuff();
+static foreach_reverse (thing; things) doStuff();
```

Comment:
This will have grammar errors after upgrading libdparse (which isn't a big problem because the unittest works, but it will eventually make the test output unreadable). Imagine the static foreach would be a static if, you can't just have function calls at a global level. Please put them in some function like `void main() {}` or change the body of the foreachs to declarations